doc/TSCScope.md

```diff
@@ -0,0 +1,31 @@
+Subject to the contraints defined in the foundation
+charter, these are the areas that the TSC has autonomy
+in making decisions (some of which can be delegated to
+other groups, for example the CTC).  Outside of these
+areas the TSC will collaborate with the board
+when making decisions:
+
+* Code/Doc/ changes to any of the repositories under github/nodejs
+* Standards and processes covering contributions to repositories under
```

Comment:
What do you think about something along the lines of `Standards and processes covering contributions to and interactions with repositories and related Internet properties under github/nodejs`? I feel that `contributions to repositories` is rather narrow and may not cover certain types of interactions, e.g. IRC, evangalism WG's interactions on non-GitHub mediums (Twitter, etc). I think these fall under our scope.